### ISSUE TITLE
[0.2.23] Crash on World Load

### ISSUE THREAD

**Issue by rramazani** [AllTheMods/ATM-9#889]
Hi,

When loading the world after the recent update, Minecraft crashes with the following error. It seems like the problem is in EnderIO energy conduit?

```
---- Minecraft Crash Report ----
// Why did you do that?

Time: 2023-11-26 12:47:40
Description: Exception in server tick loop

java.util.NoSuchElementException: No value present
	at java.util.Optional.orElseThrow(Optional.java:377) ~[?:?] {re:mixin}
	at com.enderio.conduits.common.types.EnergyConduitTicker.tickGraph(EnergyConduitTicker.java:36) ~[EnderIO-1.20.1-6.0.22-alpha.jar%23665!/:6.0.22-alpha] {re:classloading}
	at com.enderio.api.conduit.ticker.ILoadedAwareConduitTicker.tickGraph(ILoadedAwareConduitTicker.java:27) ~[EnderIO-1.20.1-6.0.22-alpha.jar%23665!/:6.0.22-alpha] {re:classloading}
	at com.enderio.conduits.common.network.ConduitSavedData.tick(ConduitSavedData.java:288) ~[EnderIO-1.20.1-6.0.22-alpha.jar%23665!/:6.0.22-alpha] {re:classloading}
	at com.enderio.conduits.common.network.ConduitSavedData.onLevelTick(ConduitSavedData.java:272) ~[EnderIO-1.20.1-6.0.22-alpha.jar%23665!/:6.0.22-alpha] {re:classloading}
	at com.enderio.conduits.common.network.__ConduitSavedData_onLevelTick_LevelTickEvent.invoke(.dynamic) ~[EnderIO-1.20.1-6.0.22-alpha.jar%23665!/:6.0.22-alpha] {re:classloading,pl:eventbus:B}
	at net.minecraftforge.eventbus.ASMEventHandler.invoke(ASMEventHandler.java:73) ~[eventbus-6.0.5.jar%2387!/:?] {}
	at net.minecraftforge.eventbus.EventBus.post(EventBus.java:315) ~[eventbus-6.0.5.jar%2387!/:?] {}
	at net.minecraftforge.eventbus.EventBus.post(EventBus.java:296) ~[eventbus-6.0.5.jar%2387!/:?] {}
	at net.minecraftforge.event.ForgeEventFactory.onPostLevelTick(ForgeEventFactory.java:930) ~[forge-1.20.1-47.2.0-universal.jar%23931!/:?] {re:mixin,re:classloading,pl:mixin:A}
	at net.minecraft.server.MinecraftServer.m_5703_(MinecraftServer.java:899) ~[client-1.20.1-20230612.114412-srg.jar%23926!/:?] {re:mixin,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:mixin:A}
	at net.minecraft.server.MinecraftServer.m_5705_(MinecraftServer.java:814) ~[client-1.20.1-20230612.114412-srg.jar%23926!/:?] {re:mixin,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:mixin:A}
	at net.minecraft.client.server.IntegratedServer.m_5705_(IntegratedServer.java:89) ~[client-1.20.1-20230612.114412-srg.jar%23926!/:?] {re:mixin,pl:runtimedistcleaner:A,re:classloading,pl:mixin:APP:modernfix-common.mixins.json:perf.thread_priorities.IntegratedServerMixin,pl:mixin:A,pl:runtimedistcleaner:A}
	at net.minecraft.server.MinecraftServer.m_130011_(MinecraftServer.java:661) ~[client-1.20.1-20230612.114412-srg.jar%23926!/:?] {re:mixin,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:mixin:A}
	at net.minecraft.server.MinecraftServer.m_206580_(MinecraftServer.java:251) ~[client-1.20.1-20230612.114412-srg.jar%23926!/:?] {re:mixin,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:mixin:A}
	at java.lang.Thread.run(Thread.java:833) ~[?:?] {re:mixin}
```

**Comment by TigerWalts** [comment]
It looks like a recent commit to EnderIO may fix this:

- https://github.com/Team-EnderIO/EnderIO/commit/a02ae18079be68c2b9c2268fffa02c4dde698be1

**Comment by flexpex** [comment]
Im having the same issue, cant seem to make any version other than .21 work, i had version .22 work one time after removing some of the conduits but then it went back to crashing on loading into the world.

**Comment by MrSnippy10** [comment]
Has there Been any fix yet?
i have yet to find a fix.


My crash for context
[https://pastebin.com/kxxE8CNR]

**Comment by flexpex** [comment]
> Has there Been any fix yet? i have yet to find a fix.
> 
> My crash for context [https://pastebin.com/kxxE8CNR]

i fixed it by simply launching into the world with update 0.21 then removed all the energy conduits and then i updated to 0.25 and it eventually worked. took a few tries to find all the conduits tho.
Now i simply avoid using energy conduits, instead im mostly using Flux networks.

**Comment by ADerpynarwhal** [comment]
Fixed in 1.21.1-9, per mod creator.
